cli/azd/cmd/deploy.go

```diff
@@ -154,7 +154,7 @@ func (d *deployAction) Run(ctx context.Context, cmd *cobra.Command, args []strin
 			deployMsg := fmt.Sprintf("Deploying service %s...", output.WithHighLightFormat(svc.Config.Name))
 			console.Message(ctx, deployMsg)
 
-			spinner, ctx := spin.GetOrCreateSpinner(ctx, deployMsg)
+			spinner, ctx := spin.GetOrCreateSpinner(ctx, cmd.OutOrStdout(), deployMsg)
```

Comment:
I think I might be missing something... but shouldn't this be using the `ctx.GetWriter` that is essentially `cmd.OutOrStdout()` except with coloring fixed?

Reply:
Yeah - this would be better - and likely sets you up for success in your efforts (since we just worry about injecting a `writer` in the future) - I will make the change.

Reply:
@weikanglim Made the change here - thanks for the feedback.